semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -434,8 +526,26 @@ object Main {
       }
     }
 
-    try loop(sym.infoType)
-    catch {
+    try {
+      if (sym.isAlias) {
```

Comment:
Looks like the logic inside this try body can be extracted into a method. I like it best when the try body is as small as possible `try process(foo) catch ...`

Reply:
Could you elaborate on the reasons? Is this a purely stylistic preference, or this also influences IDE support and/or runtime performance?

Reply:
Stylistic reasons, I get uneasy with massive try/catch blocks